components/post_view/post_attachment_opengraph/post_attachment_opengraph.jsx

```diff
@@ -205,20 +166,17 @@ export default class PostAttachmentOpenGraph extends React.PureComponent {
             props,
         });
 
-        const {data} = await this.props.actions.editPost(patchedPost);
-        if (data) {
-            if (this.mounted) {
-                this.setState({removePreview: true});
-            }
-        }
+        return this.props.actions.editPost(patchedPost);
     }
 
-    isRemovePreview(post) {
-        if (post && post.props) {
-            return post.props[PostTypes.REMOVE_LINK_PREVIEW] && post.props[PostTypes.REMOVE_LINK_PREVIEW] === 'true';
+    hasPreviewBeenRemoved() {
+        const {post} = this.props;
+
+        if (!post || !post.props) {
```

Comment:
Did we ever enable the setting to allow something like (can't recall the exact syntax):
```js
return post?.props?.[PostTypes.REMOVE_LINK_PREVIEW] === 'true'
```

Reply:
I don't think we ever enabled that on the web app. That is the correct syntax though.